packages/block-editor/src/components/multi-select-scroll-into-view/index.js

```diff
@@ -21,19 +21,29 @@ import { getBlockDOMNode } from '../../utils/dom';
  */
 export default function MultiSelectScrollIntoView() {
 	const selector = ( select ) => {
-		const { getBlockSelectionEnd, isMultiSelecting } = select(
-			'core/block-editor'
-		);
+		const {
+			getBlockSelectionEnd,
+			getBlockSelectionStart,
+			isMultiSelecting,
+		} = select( 'core/block-editor' );
 
 		return {
 			selectionEnd: getBlockSelectionEnd(),
+			selectionStart: getBlockSelectionStart(),
 			isMultiSelecting: isMultiSelecting(),
 		};
 	};
-	const { selectionEnd, isMultiSelecting } = useSelect( selector, [] );
+	const { selectionStart, selectionEnd, isMultiSelecting } = useSelect(
+		selector,
+		[]
+	);
 
 	useEffect( () => {
-		if ( ! selectionEnd || isMultiSelecting ) {
+		if (
+			! selectionEnd ||
+			isMultiSelecting ||
+			selectionEnd === selectionStart
```

Comment:
How about using the `hasMultiSelection` selector?

Reply:
yes, this works too

Reply:
Updated